internal/target/tcp/register.go

```diff
@@ -18,6 +19,7 @@ type targetHooks struct{}
 
 func init() {
 	target.Register(Subtype, targetHooks{}, TargetPrefix)
+	globals.RegisterPrefixSubtype(globals.TcpTargetPrefix, Subtype)
```

Comment:
Will you follow up with an enterprise PR after merge?

Reply:
Yes